core/src/com/unciv/ui/victoryscreen/VictoryScreen.kt

```diff
@@ -280,4 +289,17 @@ class VictoryScreen(val worldScreen: WorldScreen) : PickerScreen() {
         civGroup.pack()
         return civGroup
     }
+
+    private fun getRankingColumnIcon(type: RankingType): Image? {
```

Comment:
Just FYI: Some may not like it, but that Image getter could be a "property" of the enum... Either as property containing a lambda -
```kotlin
enum class RankingType(val getImage: ()->Image?) {
    Score({ ImageGetter.getImage("OtherIcons/Cultured").apply { color = Color.FIREBRICK } }),
    ...
```
..or as normal method..
```kotlin
enum class RankingType {
    Score {
        override fun getImage() =
                ImageGetter.getImage("OtherIcons/Cultured").apply { color = Color.FIREBRICK }
    },
    ...
    ;
    open fun getImage(): Image? = null
}
```
both then used as `category.getImage()`. No need for a switch statement. Matter of taste.

Reply:
I chose the first approach.